guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -323,17 +286,26 @@ php bin/magento setup:upgrade
 php bin/magento maintenance:disable
 ```
 
-### (Optional) Restart Varnish
+### Restart Varnish
 
-Restart Varnish if you use it for page caching.
+_Optional_—If you use Varnish for page caching, restart it:
```

Comment:
wrap the dash in spaces

Reply:
This formats correctly, using an em dash: 
![screen shot 2018-09-13 at 9 43 19 am](https://user-images.githubusercontent.com/14031342/45495837-b60b1a80-b739-11e8-8831-bb09297e44e4.png)